statusHandler/statusMetricsProvider.go

```diff
@@ -0,0 +1,87 @@
+package statusHandler
+
+import (
+	"sync"
+)
+
+// StatusMetricsProvider will handle displaying at /node/details all metrics already collected for other status handlers
+type StatusMetricsProvider struct {
```

Comment:
Can you please rename? this sounds like an interface

Reply:
done